client/src/models/User.js

```diff
@@ -204,7 +204,8 @@ class User {
 function toProperCase(name) {
     if (!name) return;
     const lower = name.toLowerCase();
-    return lower.charAt(0).toUpperCase() + lower.slice(1);
+    // return lower.charAt(0).toUpperCase() + lower.slice(1);
+    return lower
```

Comment:
The serialize fix is right--toLowerCase(). Why do we want to change the function toProperCase to return lowercase strings?